src/version.h

```diff
@@ -10,7 +10,7 @@
  * network protocol versioning
  */
 
-static const int PROTOCOL_VERSION = 170120;
+static const int PROTOCOL_VERSION = 170130;
```

Comment:
This is the one thing where I can never remember how it works. We need a comment associated with this constant that references the documentation as to how it's set.

Reply:
Any time the P2P network protocol changes, we bump this. Consensus rule changes are a subset of those, and also what we use to bump un-upgraded peers off our connection list. So we set this equal to the `nProtocolVersion` from the network params of the network and upgrade we are setting the activation height for.